go/vt/sqlparser/parse_test.go

```diff
@@ -2639,6 +2639,10 @@ func TestKeywords(t *testing.T) {
 		}, {
 			input:  "select /* unused keywords as cols */ write, virtual from t where trailing = 'foo'",
 			output: "select /* unused keywords as cols */ `write`, `virtual` from t where `trailing` = 'foo'",
+		}, {
+			input: "insert into x (status) values (42)",
```

Comment:
Add a test for update and delete too, bet one of them has the same issue.

Also you or Vinai should expand the reserved word test you wrote to include updates, deletes, inserts